### PR TITLE
Developement

### PR DESCRIPTION
**Pull request by thankgodr**
- Added change listener to each task
- Change completed to true or false 
- store to Local storage on change

### CODE REVIEW

File: src/js/index.js [hunk 1]
```diff
@@ -40,6 +40,22 @@ function printInitialTasks() {
...
     checkbox.style.marginRight = '17px';
     checkbox.value = task.completed;
+    if (task.completed) {
+      checkbox.checked = true;
+    }
+    checkbox.addEventListener('change', () => {
+      if (checkbox.checked) {
+        console.log('ischecked');
+        task.completed = true;
+        taskManager.updateALL();
```
Comment (mikenath223) [inline src/js/index.js:50]: - [x] Kindly change this and all other functions to es6 arrow function syntax, as stated in the JavaScript best practices guide.

**Comment by mikenath223** [comment]
- [x] In the readme, please include the getting started guide following [this template](https://github.com/microverseinc/readme-template).

**Review by mikenath223 — CHANGES_REQUESTED** [review]
Hello  

You've done pretty well so far!
But for a few issues, you are almost done with this. Hang in there!


## Required Changes ♻️
_Please check the comments under the review._

Happy coding, Cheers! 🚀🚀

Do feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, whether it is valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Kindly refer to the inline comments above this comment for the review.**

**Review by sinansevgi — APPROVED** [review]
# Approved :heavy_check_mark: 

Hi @thankgodr , 

Great work on making changes requested by the previous code reviewer.
Your project is complete!  
Congratulations! 🎉

You can merge this pull request and continue with your journey :wave: 


Happy Hacking! :computer: 

-----

You can reply to this conversation if you have any questions or need elaboration. :wave: 
Sinan Sevgi


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/js/index.js (hunk 1 above)
Comment (sinansevgi) [reply src/js/index.js:50]: :heavy_check_mark: well resolved

**Pull request merged**